db/migrate/20231129103914_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] The `text` column should of type `t.text`, not `t.string`. Kindly fix this or run a new migration file to fix it in the schema.rb file